internal/app/coreValues/service.go

```diff
@@ -26,35 +29,42 @@ func NewService(coreValuesRepo repository.CoreValueStorer) Service {
 	}
 }
 
-func (cs *service) ListCoreValues(ctx context.Context) (resp []dto.ListCoreValuesResp, err error) {
+func (cs *service) ListCoreValues(ctx context.Context) (resp []dto.CoreValue, err error) {
 
-	resp, err = cs.coreValuesRepo.ListCoreValues(ctx)
+	dbResp, err := cs.coreValuesRepo.ListCoreValues(ctx)
 	if err != nil {
-		logger.WithField("err", err.Error()).Error("Error while fetching data")
 		err = apperrors.InternalServerError
 	}
 
+	fmt.Println(dbResp)
+
+	for i := 0; i < len(dbResp); i++ {
```

Comment:
can use for _, resp := range dbResp, looks better.

Reply:
Done